setup.py

```diff
@@ -79,7 +79,9 @@ def finalize_options(self):
         'dimod/include/',
         ],
     install_requires=[
-        'numpy>=1.20.0,<2.0.0',
+        # this is the oldest supported NumPy on Python 3.8, we cannot
+        # support the oldest for 3.7, 1.14.5
+        'numpy>=1.17.3,<2.0.0',
```

Comment:
I also think we should lift the `2.0` upper bound, but we can discuss that and/or do it in a separate PR.

Reply:
Agree it's a bit pointless, we should probably either lift it altogether or tighten it down to 1.n+2.0. But yeah, followup PR.